python/ray/rllib/train.py

```diff
@@ -29,16 +29,24 @@
     epilog=EXAMPLE_USAGE)
 
 # See also the base parser definition in ray/tune/config_parser.py
-parser.add_argument("--redis-address", default=None, type=str,
-                    help="The Redis address of the cluster.")
-parser.add_argument("--num-cpus", default=None, type=int,
-                    help="Number of CPUs to allocate to Ray.")
-parser.add_argument("--num-gpus", default=None, type=int,
-                    help="Number of GPUs to allocate to Ray.")
-parser.add_argument("--experiment-name", default="default", type=str,
-                    help="Name of experiment dir.")
-parser.add_argument("-f", "--config-file", default=None, type=str,
-                    help="If specified, use config options from this file.")
+parser.add_argument(
```

Comment:
Question: How are you splitting the placement of these args? From what I understand there's some overlap between `tune` and `train` but it's not entirely clear what/why...

Reply:
Currently only the ray-specific ones are duplicated. RLlib also has a few extras like --env and --experiment-name for convenience.